CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib

```diff
@@ -25,6 +25,845 @@ @online{SKA
+      eprint         = "1410.3012",
+      archivePrefix  = "arXiv",
+      primaryClass   = "hep-ph",
+      reportNumber   = "LU-TP-14-36, MCNET-14-22, CERN-PH-TH-2014-190,
+                        FERMILAB-PUB-14-316-CD, DESY-14-178, SLAC-PUB-16122",
+      SLACcitation   = "%%CITATION = ARXIV:1410.3012;%%"
+}
+
+@article{Green:2012gv,
+      author         = "Green, C. and Kowalkowski, J. and Paterno, M. and
+                        Fischler, M. and Garren, L. and Lu, Q.",
+      title          = "{The Art Framework}",
+      booktitle      = "{Proceedings, 19th International Conference on Computing
+                        in High Energy and Nuclear Physics (CHEP 2012): New York,
+                        USA, May 21-25, 2012}",
+      journal        = "J. Phys. Conf. Ser.",
+      volume         = "396",
+      year           = "2012",
+      pages          = "022020",
+      doi            = "10.1088/1742-6596/396/2/022020",
+      reportNumber   = "FERMILAB-PUB-12-301-CD",
+      SLACcitation   = "%%CITATION = 00462,396,022020;%%"
+}
+
+@article{Heck:1998vt,
+      author         = "Heck, D. and Schatz, G. and Thouw, T. and Knapp, J. and
+                        Capdevielle, J. N.",
+      title          = "{CORSIKA: A Monte Carlo code to simulate extensive air
+                        showers}",
+      year           = "1998",
+      reportNumber   = "FZKA-6019",
+      SLACcitation   = "%%CITATION = FZKA-6019;%%"
+}
+
+@article{1748-0221-6-10-P10002,
+  author={M Szydagis and N Barry and K Kazkaz and J Mock and D Stolp and M Sweany and M Tripathi and S Uvarov and N Walsh and M Woods},
+  title={NEST: a comprehensive model for scintillation yield in liquid xenon},
+  journal={Journal of Instrumentation},
+  volume={6},
+  number={10},
+  pages={P10002},
+  url={http://stacks.iop.org/1748-0221/6/i=10/a=P10002},
+  year={2011},
+  abstract={A comprehensive model for explaining scintillation yield in liquid xenon is introduced. We unify various definitions of work function which abound in the literature and incorporate all available data on electron recoil scintillation yield. This results in a better understanding of electron recoil, and facilitates an improved description of nuclear recoil. An incident gamma energy range of O (1 keV) to O (1 MeV) and electric fields between 0 and O (10 kV/cm) are incorporated into this heuristic model. We show results from a Geant4 implementation, but because the model has a few free parameters, implementation in any simulation package should be simple. We use a quasi-empirical approach with an objective of improving detector calibrations and performance verification. The model will aid in the design and optimization of future detectors. This model is also easy to extend to other noble elements. In this paper we lay the foundation for an exhaustive simulation code which we call NEST (Noble Element Simulation Technique).}
+}
+
+@article{1742-6596-898-4-042001,
```

Comment:
This is also a duplicate, however I notice the original entry also has some strange characters in the title that need to be fixed.